src/components/Gallery/index.js

```diff
@@ -20,158 +24,108 @@ import {
-              <GallerSliderHeader>
-                {query?.title ? (
-                  <Title as="h2" type="heading2">
-                    {query.title}
-                  </Title>
-                ) : (
-                  ''
-                )}
-                <Icon type="cross-des" />
-              </GallerSliderHeader>
-              {query?.content?.length > 0 ? (
-                <GallerySliderContainer>
-                  <GallerySliderContent
-                    slidesToShow={3}
-                    slidesToScroll={2}
-                    infinite={true}
-                  >
-                    {query?.content
-                      ? query.content.map((info, index) => {
-                          const sliderImage = info?.gallery_img?.localFile
-                            ?.childImageSharp?.fluid
-                            ? info.gallery_img.localFile.childImageSharp.fluid
-                            : ''
-                          return (
-                            <Fragment key={index}>
-                              <GalleryItemContent>
-                                <GallerySliderItemsContainer>
-                                  {sliderImage ? (
-                                    <Img fluid={sliderImage} />
-                                  ) : (
-                                    ''
-                                  )}
-                                  <GallerSliderTitle>
-                                    {info?.gallery_section_title?.text ? (
-                                      //CHANGE TITLE
-                                      <Title as="h3" type="heading8">
-                                        {info.gallery_section_title.text}
-                                      </Title>
-                                    ) : (
-                                      ''
-                                    )}
-                                  </GallerSliderTitle>
-                                  <GallerySliderText>
-                                    {info?.gallery_section_text ? (
-                                      <Text type="smallText700">
-                                        {info.gallery_section_text}
-                                      </Text>
-                                    ) : (
-                                      ''
-                                    )}
-                                  </GallerySliderText>
-                                </GallerySliderItemsContainer>
-                                <GallerySliderIcon>
-                                  <Icon type="flower" />
-                                </GallerySliderIcon>
-                              </GalleryItemContent>
-                            </Fragment>
-                          )
-                        })
-                      : ''}
-                  </GallerySliderContent>
-                </GallerySliderContainer>
+            <GallerSliderHeader>
+              {title ? (
```

Comment:
title check should probably wrap the GallerSliderHeader. Also, their is a grammar here in your component name